### PR TITLE
Minor fixes to fit ci update

### PR DESCRIPTION
**Pull request by LeoFVO**
* Fix lint for url
* Fix ci for doc lint

### CODE REVIEW

**Comment by LeoFVO** [comment]
Hey @bee-san, I made some changes du to my last contribution, now ci is working perfectly.
Can you review it please :)

File: src/input.rs [hunk 1]
```diff
@@ -68,8 +68,8 @@ fn parse_range(input: &str) -> Result<PortRange, String> {
 /// Fast Port Scanner built in Rust.
 /// WARNING Do not use this program against sensitive infrastructure since the
 /// specified server may not be able to handle this many socket connections at once.
-/// - Discord https://discord.gg/GFrQsGy
-/// - GitHub https://github.com/RustScan/RustScan
+/// - Discord <https://discord.gg/GFrQsGy>
```
Comment (bee-san) [inline src/input.rs:71]:
```suggestion
/// - Discord <http://discord.skerritt.blog>
```

Annoyingly we have updated our Discord, and also there is no SSL on that domain _yet_ as I have run out of CLoudflare page-rules 🙈 it is on my todo list

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -87,8 +87,6 @@ jobs:
 
       - name: Run rustdoc lints
         uses: actions-rs/cargo@v1
-        env:
-          RUSTDOCFLAGS: '-D missing_docs -D rustdoc::missing_doc_code_examples'
```
Comment (bee-san) [inline .github/workflows/build.yml:91]: Thanks for this, I'll add docs eventually 👍🏻  😂

**Review by bee-san — CHANGES_REQUESTED** [review]

**Comment by LeoFVO** [comment]
All threads resolved :+1:

File: .github/workflows/build.yml (hunk 2 above)
Comment (LeoFVO) [reply .github/workflows/build.yml:91]: It's Okay for now, only doc for macro were blocking

**Review by bee-san — APPROVED** [review]

**Comment by bee-san** [comment]
🥳  thanks!!

**Pull request merged**